### ISSUE TITLE
Tidex loadMarkets() error

### ISSUE THREAD

**Issue by ndubel** [ccxt/ccxt#4146]
- Programming Language: JS
- CCXT version: 1.17.449
- Exchange: Tidex
- Method: loadMarkets()

When I try to create instance of tidex i catch an error:

```
fetch:
 tidex GET https://api.tidex.com/api/3/info 
Request:
 { 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36' } 
 undefined 

handleRestResponse:
 tidex GET https://api.tidex.com/api/3/info 200 OK 
Response:
 { Date: 'Fri, 09 Nov 2018 10:32:06 GMT',
  'Content-Type': 'application/json; charset=utf-8',
  'Transfer-Encoding': 'chunked',
  Connection: 'close',
  'Set-Cookie': '__cfduid=de9709a4d8b4aa589923bcd05e1dd6f331541759526; expires=Sat, 09-Nov-19 10:32:06 GMT; path=/; domain=.tidex.com; HttpOnly',
  'Expect-Ct': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
  Server: 'cloudflare',
  'Cf-Ray': '476f9d0f4e179c23-AMS',
  'Content-Encoding': 'gzip' } 
 {"server_time":1541759525,"pairs":{"ltc_btc":{"decimal_places":8,"min_price":0.00000001,"max_price":3.0,"min_amount":0.001,"max_amount":1000000.0,"min_total":0.0001,"hidden":0,"fee":0.1},"eth_btc":

...

{"decimal_places":8,"min_price":0.0000001,"max_price":20.0,"min_amount":0.001,"max_amount":1000000.0,"min_total":0.001,"hidden":0,"fee":0.1}}} 

fetch:
 tidex GET https://web.tidex.com/api/currency 
Request:
 { 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36' } 
 undefined 

PROBLEM { Error: tidex GET https://web.tidex.com/api/currency request timed out (10000 ms)
    at timeout.catch.e (/var/www/node/bot/node_modules/ccxt/js/base/Exchange.js:436:27)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7) constructor: [Function: RequestTimeout] }

```

There is no anything about web.tidex.com in tidex api https://tidex.com/exchange/public-api, but in works fine until yesterday

**Comment by kroitor** [comment]
> There is no anything about web.tidex.com in tidex api

Right, but there was *something* about web.tidex.com on their website, until they decided to redesign their domain structure, so, it has been renamed to gate.tidex.com and it is a bit broken since. We'll add a fix for it shortly.

We had to use their web endpoints (as opposed to API endpoints) because upon the moment of integration, their API/docs didn't have enough info on currencies and markets, and we had to scrape that info from their website live. Now it has the info that was missing previously, it just needs some editing in order to make it work again.
